api/data_explorer/controllers/export_url_controller.py

```diff
@@ -113,10 +111,8 @@ def _get_entities_dict(cohort_name, query, filter_arr):
             # table_name attribute. Unfortunately RAWLS doesn't allow
             # periods here. RAWLS does allow periods in attributes. So use
             # underscores here and periods in table_name attribute.
-            'name': table_name.replace('.', '_'),
-            'attributes': {
-                'table_name': table_name
-            }
+            'name': name.replace('.', '_'),
```

Comment:
Do you mind changing this to:
```
'name': name.replace('.', '_').replace(':', '_'),
```

This fixes export for my test project, which has "google.com:" in it. Thanks.

Reply:
Done.